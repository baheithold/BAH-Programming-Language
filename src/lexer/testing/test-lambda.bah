# This file tests valid lambda


function f() {
    return lambda(x, y) {
        x + y;
    };
}
