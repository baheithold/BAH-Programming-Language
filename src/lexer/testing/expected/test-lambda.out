FUNCTION_KEYWORD
ID_TYPE f
OPAREN
CPAREN
OBRACE
RETURN_KEYWORD
LAMBDA_KEYWORD
OPAREN
ID_TYPE x
COMMA
ID_TYPE y
CPAREN
OBRACE
ID_TYPE x
PLUS_BINARY
ID_TYPE y
SEMICOLON
CBRACE
SEMICOLON
CBRACE
END_OF_INPUT
